src/v2/Apps/Home/__tests__/HomeTroveRail.jest.tsx

```diff
@@ -0,0 +1,90 @@
+import { graphql } from "react-relay"
+import { useTracking } from "react-tracking"
+import { setupTestWrapper } from "v2/DevTools/setupTestWrapper"
+import { HomeTroveArtworksRailFragmentContainer } from "../Components/HomeTroveArtworksRail"
+
+jest.unmock("react-relay")
+
+const { getWrapper } = setupTestWrapper({
+  Component: HomeTroveArtworksRailFragmentContainer,
+  query: graphql`
+    query HomeTroveArtworksRail_Test_Query @relay_test_operation {
+      viewer {
+        ...HomeTroveArtworksRail_viewer
+      }
+    }
+  `,
+})
+
+describe("HomeTroveArtworksRail", () => {
+  const trackEvent = jest.fn()
+
+  beforeEach(() => {
+    ;(useTracking as jest.Mock).mockImplementation(() => ({ trackEvent }))
+  })
+
+  afterEach(() => {
+    trackEvent.mockClear()
+  })
+
+  it("renders correctly", () => {
+    const wrapper = getWrapper({
+      Viewer: () => ({
+        artworksConnection: {
+          edges: [
+            {
+              node: {
+                title: "Test Artwork",
+                href: "test-href",
+              },
+            },
+          ],
+        },
+      }),
+    })
+
+    expect(wrapper.text()).toContain("Test Artwork")
+    expect(wrapper.html()).toContain("test-href")
+  }),
+    it("tracks artwork click", () => {
+      const wrapper = getWrapper({
+        Viewer: () => ({
+          artworksConnection: {
+            edges: [
+              {
+                node: {
+                  title: "Test Artwork",
+                  href: "test-href",
+                },
+              },
+            ],
+          },
+        }),
+      })
+
+      wrapper.find("Shelf").find("RouterLink").first().simulate("click")
+      expect(trackEvent).toBeCalledWith({
```

Comment:
Not sure if damon showed you this trick, but to get these values I always just do 

```tsx
expect(trackEvent).toBeCalledWith("whatevs")
```

and then copy the failed output from Jest